common/models/application.js

```diff
@@ -71,13 +72,11 @@ function generateKey(hmacKey, algorithm, encoding) {
 
 module.exports = function(Application) {
   // Workaround for https://github.com/strongloop/loopback/issues/292
-  Application.definition.rawProperties.created.default =
```

Comment:
> ```
>> Mocha exploded!
>> TypeError: Cannot create property 'default' on string 'date'
>>   at module.exports (/Users/loay/Desktop/NodeEE/loopback/common/models/application.js:75:56)
```

Ah, makes sense now.

> ```
 "created": {
      "default": "date"
    },
    "modified": {
      "default": "date"
    }
```

The correct solution is to use `"defaultFn": "now"`, see https://github.com/strongloop/loopback-datasource-juggler/pull/458